test/stability-tests/src/it/java/software/amazon/awssdk/stability/tests/s3/S3AsyncStabilityTest.java

```diff
@@ -49,8 +45,13 @@ public static void cleanup() {
     }
 
     @Test
-    @Order(1)
-    public void putObject() {
+    @Override
+    public void putObject_getObject() {
```

Comment:
Any reason for merging both APIs into single test?

Reply:
Because `getObject` depends on `putObject` and this way the test would fail if `getObject` fails.